js/showPassword.js

```diff
@@ -0,0 +1,18 @@
+const showPassword = (tag) => {
+  const eyeIcon = tag;
```

Comment:
이 함수가 호출되기 전에 조건문으로 제한하고 있군요.

어찌되었건 `showPassword`에서 기대하는 파라미터는 eyeIcon니까 그대로 이름지어주시면 의도를 보여줄 수 있습니다. 

그러면 `showPassword(eyeIconElement)` 이런 형태니까, 패스워드를 보여주는 함수인데, 눈모양 아이콘요소를 전달해야한다. 는 뜻이겠죠. 그러면 이제 함수이름을 검토할 차례입니다.
함수 내용 중 패스워드를 보여주는 지와 관련된 내용은 아래 8L/13L이군요.
나머지 내용은 눈모양아이콘을 바꾸는 내용이고요.

위 내용을 종합해서 함수시그니처를 제안드려본다면 이렇습니다.

```suggestion
const togglePasswordVisibility = (eyeIconElement) => {
```